bin/tidyall

```diff
@@ -54,6 +54,7 @@ if ( ( $all_files || $svn_files || $git_files ) ) {
     die "cannot use filename(s) with -a/--all, -s/--svn, or -g/--git"
       if @ARGV;
     $conf_file ||= $tidyall_class->find_conf_file( cwd() );
+    $params{root_dir} = cwd() if $conf_file eq "$ENV{HOME}/.tidyall.ini";
```

Comment:
This is just wrong, since the file is either `tidyall.ini` (no dot) or `.tidyallrc`. You'd need to check for both.

Better to check something like:

```perl
... if dirname($conf_file) eq $ENV{HOME};
```